src/v0/util/index.test.js

```diff
@@ -506,3 +506,50 @@ describe('validateEventAndLowerCaseConversion Tests', () => {
     }).toThrow(InstrumentationError);
   });
 });
+
+describe('extractCustomFields', () => {
+  // Handle reserved words in message keys
+  it('should handle reserved words in message keys when keys are provided', () => {
+    const message = {
+      traits: {
+        firstName: 'John',
+        lastName: 'Doe',
+        email: 'john.doe@example.com',
+      },
+      context: {
+        traits: {
+          phone: '1234567890',
+          city: 'New York',
+          country: 'USA',
+        },
+      },
+      properties: {
+        title: 'Developer',
+        organization: 'ABC Company',
+        zip: '12345',
+        prototype: 'reserved',
```

Comment:
Add some more test cases where `prototype` is there in other places

Reply:
added test cases for all reserved key words in root , as well as inside traits , properties